src/methods.jl

```diff
@@ -329,3 +329,52 @@ function combine(frames::Vararg{<:AbstractArray}; method = median)
 end
 
 combine(frames::AbstractVector{<:AbstractArray}; method = median) = combine(frames..., method = method)
+
+
+"""
+    subtract_dark!(frame::AbstractArray, dark_frame::AbstractArray; data_exposure = 1, dark_exposure = 1, scale = false)
+
+In place version of [`subtract_dark`](@ref)
```

Comment:
```suggestion
In-place version of [`subtract_dark`](@ref)
```

Reply:
> @siddharthlal25 I've noted that you always use the same branch called `devel` for your work. I just wanted to let you know that it isn't a good practice to do so, for each feature you should create _a new branch_ (possibly with a somewhat short yet meaningful name, easy for you to identify). Using a single branch workflow doesn't let you work on multiple features at the same time, and I think it's a good idea to practice with git's features  (Also, the fact that you always use the same branch doesn't make my life particularly easy when I want to check out your work locally)

Sure, I will keep something short and meaningful! Sorry for the hassle.